include/mpi_fsi.h

```diff
@@ -7,6 +7,10 @@
 #include "mpi_fluid_solver.h"
 #include "mpi_shared_solid_solver.h"
 
+
+// for building in some systems
+#include <optional>
```

Comment:
Shouldn't this be added to `mpi_fsi.cpp` instead? I don't find usage of `std::optional` in this file.

Reply:
yes you are right, now the change is on `mpir_fsi.cpp`.